### PR TITLE
Pause state PR

### PR DESCRIPTION
**Pull request by hrishileanplum**
What              | Where/Who
------------------|----------------------------------------
JIRA Issue        | [E2-1747](https://leanplum.atlassian.net/browse/E2-1747])
People Involved   | @colleague1, @colleague2

[Notes on PR descriptions](https://leanplum.atlassian.net/wiki/spaces/PR/pages/288424408/Creating+a+GitHub+Pull+Request)

### CODE REVIEW

File: Leanplum/Leanplum.m [hunk 1]
```diff
@@ -477,14 +505,52 @@ + (void)triggerStartIssued
...
+
+    [self onStartIssued:^{
+        [self pauseStateInternal];
+    }];
+}
+
++ (void)pauseStateInternal
+{
+    [LPPauseStateApi pauseStateWithParameters:@{} success:^{
+        NSLog(@"pausedStateInternal");
```
[the remark this answers is not in the document]
Comment (hrishileanplum) [reply Leanplum/Leanplum.m]: done

File: LeanplumTests/Extensions/LPDeviceApi+Categories.m [hunk 2]
```diff
@@ -26,19 +26,6 @@ + (void)swizzle_methods
     }
 }
 
-+ (void)swizzle_methods_undo
```
[the remark this answers is not in the document]
Comment (hrishileanplum) [reply LeanplumTests/Extensions/LPDeviceApi+Categories.m]: this method was unused and is not needed. So this is deleted.

File: Leanplum/Leanplum.m [hunk 3]
```diff
@@ -576,23 +642,19 @@ + (void)didReceiveRemoteNotification:(NSDictionary *)userInfo
 + (void)didReceiveNotificationResponse:(UNNotificationResponse *)response
                  withCompletionHandler:(void (^)(void))completionHandler
 {
-    LP_TRY
```
[the remark this answers is not in the document]
Comment (hrishileanplum) [reply Leanplum/Leanplum.m]: I am planning to add these system wide. I have been getting compiler architecture errors. So, that i would resolve in an independent PR.

[the remark this answers is not in the document]
Comment (hrishileanplum) [reply Leanplum/Leanplum.m]: https://leanplum.atlassian.net/browse/E2-2064

File: Leanplum/Cache/LPCache.m [hunk 4]
```diff
@@ -41,69 +41,7 @@ - (void)setFileAttributes:(NSArray<LPFileAttribute *> *)fileAttributes {
     return regions;
 }
 
-//ToDo: Implement feature based
```
Comment (e7mac) [inline Leanplum/Cache/LPCache.m]: let's put unrelated changes in separate PRs, to help better track where test coverage is headed
